<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StorageAnalyticsWidget</class>
 <widget class="QWidget" name="StorageAnalyticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>621</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="topMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="gridPage">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="statsTab">
          <attribute name="title">
           <string>Current Statistics</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="topMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="nx::client::desktop::TableView" name="statsTable">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>1</number>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="forecastTab">
          <attribute name="title">
           <string>Forecast for Full Storage Usage</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0">
           <property name="topMargin">
            <number>12</number>
           </property>
           <item>
            <widget class="nx::client::desktop::TableView" name="forecastTable">
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>1</number>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderShowSortIndicator" stdset="0">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QWidget" name="forecastControlWidget" native="true">
               <layout class="QGridLayout" name="gridLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0" rowspan="4">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Additional storage:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" rowspan="4">
                 <widget class="QDoubleSpinBox" name="extraSizeSpinBox">
                  <property name="suffix">
                   <string notr="true">&lt; TB&gt;</string>
                  </property>
                  <property name="maximum">
                   <double>1000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3" rowspan="4">
                 <widget class="QSlider" name="extraSpaceSlider">
                  <property name="minimumSize">
                   <size>
                    <width>350</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>400</number>
                  </property>
                  <property name="pageStep">
                   <number>100</number>
                  </property>
                  <property name="tracking">
                   <bool>true</bool>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="tickPosition">
                   <enum>QSlider::NoTicks</enum>
                  </property>
                  <property name="tickInterval">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2" rowspan="4">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string notr="true">0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="maxSizeLabel">
               <property name="text">
                <string notr="true">1000 TB</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="warnPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="warningLabel">
         <property name="text">
          <string>Server is offline. Statistics are not available.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::client::desktop::TableView</class>
   <extends>QTableView</extends>
   <header location="global">nx/client/desktop/common/widgets/table_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
